features/configuration_files/schema_validation.feature

```diff
@@ -0,0 +1,247 @@
+Feature: Validate schema
```

Comment:
Can we move some of what is tested here to specs and just have one or two scenarios here to check the default configuration and the failure case?